projects/cdk/directives/swipe/swipe.service.ts

```diff
@@ -25,14 +25,14 @@ export class TuiSwipeService extends Observable<TuiSwipeEvent> {
                     filter(
                         ([first, second]) =>
                             !!first.touches.length &&
-                            first.touches[0].identifier ===
-                                second.changedTouches[0].identifier,
+                            first.touches.item(0)?.identifier ===
```

Comment:
What is the benefit of using `.item()` over index access?

Reply:
fixed